content/blog/2022/12/2022-12-14-require-java-11.adoc

```diff
@@ -0,0 +1,223 @@
+with recent versions of the plugin build toolchain supporting baselines up to two (2) years old.
+This provides a high degree of flexibility for plugin maintainers and ultimately end users.
+For the reasons given on https://groups.google.com/g/jenkinsci-dev/c/pjfvsMw-EMM/m/OIw_hveUBwAJ[the developer mailing list], however,
+this level of compatibility could not be preserved throughout the transition to Java 11.
+We therefore took the decision to require a baseline of 2.361 or newer for plugins in recent releases of the plugin build toolchain,
+about a year ahead of our usual timeline.
+This unusually aggressive timeline does not represent a change in policy for the Jenkins project
+but is rather an exception to the rule to facilitate migration to a number of breaking changes in the upstream Java community.
+
+Such a transition constitutes a https://en.wikipedia.org/wiki/Flag_day_(computing)[flag day], or breaking change.
+Below are answers to frequently asked questions,
+inspired by Uma Chingunde's excellent framework for https://umach.medium.com/communicating-organizational-changes-6277e4b47cff[communicating organizational changes] and building on the excellent https://community.jenkins.io/t/solving-failing-parent-pom-upgrade-from-4-51-to-4-52-cheatsheet/4958[cheat sheet] started by Jean-Marc Meessen.
+
+=== What is the change?
+
+Beginning with https://github.com/jenkinsci/plugin-pom/releases/tag/plugin-4.52[4.52],
+the plugin build toolchain requires Java 11 or newer and Jenkins 2.361 or newer.
+
+=== When is the change effective?
+
+The change is effective as of plugin parent POM 4.52, which was released on December 1, 2022.
+
+=== Why is it happening?
+
+Due to a large number of breaking changes in the upstream Java community,
+it became impractical to support both Java 8 and Java 11 in the same build toolchain.
+For more details, see https://groups.google.com/g/jenkinsci-dev/c/pjfvsMw-EMM/m/OIw_hveUBwAJ[the developer mailing list].
+
+=== Who is affected by the change?
+
+This change affects all plugin developers,
+particularly those who receive updates to the plugin build toolchain via Dependabot pull requests.
+
+=== What action do I need to take?
+
+At a high level, three actions need to be taken, the third of which depends on the first two:
+
+. Adjust the `Jenkinsfile` to use Java 11 or newer, removing any Java 8 configuration(s).
+. Update the baseline to 2.361 or newer.
+. Update the plugin parent POM to 4.52 or newer.
+
+We will elaborate on each of these three points below.
+
+==== Adjust the `Jenkinsfile` to use Java 11 or newer
+
+First, consult the matrix of build configurations in the plugin's `Jenkinsfile`.
+The goal is to ensure the plugin is building on Java 11 and 17 and not on Java 8,
+as recommended in https://github.com/jenkinsci/archetypes/blob/master/common-files/Jenkinsfile[the latest version of the archetype]:
+
+[source,groovy]
+----
+buildPlugin(
+  useContainerAgent: true,
```

Comment:
Is there a reason you aren't maintaining the comment from the archetypes / other docs? I can't see anything explicitly mentioning that any tests with docker won't work with this recommended configuration
```suggestion
  useContainerAgent: true, // Set to `false` if you need to use Docker for containerized tests
```

(if above is too long then):
```suggestion
   // Set to `false` if you need to use Docker for containerized tests
  useContainerAgent: true,
```